lib/kitchen/transport/winrm.rb

```diff
@@ -452,8 +477,29 @@ def load_needed_dependencies! # rubocop:disable Metrics/MethodLength
       # Load WinRM::Transport code.
       #
       # @api private
+      def host_os_windows?
+        case RbConfig::CONFIG["host_os"]
+        when /darwin/
+          false
+        when /mswin|msys|mingw|cygwin|bccwin|wince|emc/
+          true
+        when /linux/
+          false
+        else
+          false
+        end
```

Comment:
I'd say you're okay with the super simple case of:

```ruby
case RbConfig::CONFIG["host_os"]
when /mswin|msys|mingw|cygwin|bccwin|wince|emc/
  true
else
  false
end
```

That way we're focused on the whilelisting tokens and dropping anything else by default.